src/java/org/apache/cassandra/io/sstable/format/SSTableReaderBuilder.java

```diff
@@ -226,17 +234,29 @@ void buildSummaryAndBloomFilter(boolean recreateBloomFilter,
         }
     }
 
-    /**
-     * Load bloom filter from Filter.db file.
-     *
-     * @throws IOException
-     */
-    IFilter loadBloomFilter() throws IOException
+    public static IFilter loadBloomFilter(Path path, boolean oldFormat)
```

Comment:
Actually it was intentional. I wanted to make sure it is a pure utility method (and as such it does not refer to any instance fields/methods) as when we rebase `ds-trunk` over `STAR-99`, it will be used outside of `SSTableReaderBuilder`